airbyte-integrations/connectors/source-zendesk-sunshine/metadata.yaml

```diff
@@ -1,24 +1,28 @@
 data:
+  allowedHosts:
+    hosts:
+      - ${subdomain}.zendesk.com
+  registries:
+    oss:
+      enabled: true
+    cloud:
+      enabled: true
   connectorSubtype: api
   connectorType: source
   definitionId: 325e0640-e7b3-4e24-b823-3361008f603f
-  dockerImageTag: 0.1.1
+  dockerImageTag: 0.2.1
   dockerRepository: airbyte/source-zendesk-sunshine
   githubIssueLabel: source-zendesk-sunshine
   icon: zendesk-sunshine.svg
   license: MIT
   name: Zendesk Sunshine
-  registries:
-    cloud:
-      enabled: true
-    oss:
-      enabled: true
+  releaseDate: TODO
```

Comment:
Needs to be set to a date with the format of `YYYY-MM-DD`. (today's date is fine)